src/shared/mount-util.h

```diff
@@ -11,18 +11,18 @@
 #include "errno-util.h"
 #include "macro.h"
 
-/* 4MB for contents of regular files, 128k inodes for directories, symbolic links and device specials, using
- * large storage array systems as a baseline */
-#define TMPFS_LIMITS_DEV             ",size=4m,nr_inodes=128k"
+/* The limit used for /dev in private namespaces. 4MB for contents of regular files, 64k inodes for
+ * directories, symbolic links and device specials, using large storage array systems as a baseline */
+#define TMPFS_LIMITS_PRIVATE_DEV     ",size=4m,nr_inodes=64k"
 
 /* Very little, if any use expected */
 #define TMPFS_LIMITS_EMPTY_OR_ALMOST ",size=4m,nr_inodes=1k"
 #define TMPFS_LIMITS_SYS             TMPFS_LIMITS_EMPTY_OR_ALMOST
 #define TMPFS_LIMITS_SYS_FS_CGROUP   TMPFS_LIMITS_EMPTY_OR_ALMOST
 
-/* On an extremely small device with only 256MB of RAM, 20% of RAM should be enough for the re-execution of
- * PID1 because 16MB of free space is required. */
-#define TMPFS_LIMITS_RUN             ",size=20%,nr_inodes=800k"
+/* The limit used for /run in private namespaces. On an extremely small device with only 256MB of RAM, 20% of
+ * RAM should be enough for the re-execution of PID1 because 16MB of free space is required. */
```

Comment:
The discussion about re-execution of PID1 isn't applicable to private `/run`. Maybe the limits could be tightened so that one service can't take 20% of RAM but perhaps 10% or 5%.